packages/financial-templates-lib/src/logger/Transports.ts

```diff
@@ -45,7 +45,11 @@ export function createTransports(transportsConfig: TransportsConfig = {}): Trans
     const { LoggingWinston } = require("@google-cloud/logging-winston");
     transports.push(new LoggingWinston());
     if (!require("@google-cloud/trace-agent").get().enabled) require("@google-cloud/trace-agent").start();
-  } else if (transportsConfig.createConsoleTransport != undefined ? transportsConfig.createConsoleTransport : true) {
+  } else if (
+    transportsConfig.createConsoleTransport != undefined
```

Comment:
Is this the desired condition or am I missing some defaulting logic where undefined is assumed to be true or something?
```ts
} else if (transportsConfig.createConsoleTransport || process.env.LOG_TO_CONSOLE == "true") {
```

Reply:
Ye, you're right. this can be simplified to what you showed. I made this PR during a meeting trying to solve a bug running the relayers in prod :) 

thanks for the callout. I should have marked this PR as draft :)

Reply:
it actually needs to be like this:
```
} else if (transportsConfig?.createConsoleTransport || process.env.LOG_TO_CONSOLE == "true") {
```

to re-implement the same functionality as what I had before:
1) if `createConsoleTransport` is defined then use `createConsoleTransport` value else, if `LOG_TO_CONSOLE` then true